app/partials/project-card.cjsx

```diff
@@ -31,15 +34,24 @@ ProjectCard = React.createClass
     else
       '/projects/' + @props.project.slug
 
-    <FlexibleLink to={href}>
-      <div className="project-card" ref="projectCard" style={conditionalStyle}>
+    cardBody =
+      <div className="project-card #{this.props.customCSS}" ref="projectCard" style={conditionalStyle}>
         <svg viewBox="0 0 2 1" width="100%"></svg>
-        <div className="details">
+        <div className="details" style={{cursor: "default"}}>
           <div className="name"><span>{@props.project.display_name}</span></div>
+          {<hr /> if @props.landingPage}
           {<div className="description">{@props.project.description}</div> if @props.project.description?}
           <button type="button" tabIndex="-1" className="standard-button card-button"><Translate content={"projectsPage.button"} /></button>
+          {if @props.landingPage
+            <Link to={href} className="primary-button" type="button">View Project</Link>}
         </div>
       </div>
-    </FlexibleLink>
+
+    if !@props.landingPage
+      <FlexibleLink to={href}>
+        {cardBody}
+      </FlexibleLink>
+    else
+        cardBody
```

Comment:
Yes, the cards are visible on a different PR, but I included them in this root PR since they would be included in multiple other PRs branched off of this. Making the entire image a link would be much cleaner in the long run. I can add that `border-radius` change to this PR which would close that issue.